build/webpack.config.dev.js

```diff
@@ -16,6 +17,17 @@ module.exports = (env) => {
     },
     module: {
       rules: [
+        {
+          test: /\.(woff(2)?|ttf|eot|svg)(\?v=\d+\.\d+\.\d+)?$/,
```

Comment:
Not entirely sure what is going on here.  Seems like this part loads fonts but the regex is not clear.  Could use a comment to explain.

Reply:
Yep, this is related to font parsing. It does some weird things to parse various font files and other extensions that might be appended to the end of the file. I remember getting this off of some `webpack ` documentation, but I don't remember exactly where. It ended up on another one of my `webpack` projects though, and I took it from there and put it here.